locopy/database.py

```diff
@@ -220,7 +221,15 @@ def to_dataframe(self, size=None):
 
         if len(fetched) == 0:
             return None
-        return pandas.DataFrame(fetched, columns=columns)
+
+        if df_type == "pandas":
+            import pandas
```

Comment:
I think the mid-level import is legacy, if pandas and polars are top-level dependencies, I think we can just move them to the top of the file.

Reply:
got it